client/checkout/classic/upe-deferred-intent-creation/payment-processing.js

```diff
@@ -218,6 +266,7 @@ export async function mountStripePaymentElement( api, domElement ) {
 	const upeElement =
 		gatewayUPEComponents[ paymentMethodType ].upeElement ||
 		( await createStripePaymentElement( api, paymentMethodType ) );
+	maybeEnableStripeLink( api, paymentMethodType );
```

Comment:
`maybeEnableStripeLink` will have to stay inside `payment-processing.js`, because it requires access to the Stripe Elements object that is only available therein. However, I've refactored the rest tailored to your suggestion and moved the declaration of `maybeEnableStripeLink` to `event-handlers.js`, as you've suggested. See 2ee6b6ebea00cc109bfa8c8c52069e2c2eb633a3.